.cico/cico_build_nightly.sh

```diff
@@ -28,3 +28,4 @@ build ./build/CI/Dockerfile
 set_nightly_tag
 build_and_push ./build/metadata che-plugin-metadata-broker
 build_and_push ./build/artifacts che-plugin-artifacts-broker
+build_and_push_development_container che-plugin-broker-dev
```

Comment:
I wonder why do we need che-plugin-broker-dev container? 
AFAIS it contains only godep, golint, are not they typical packages for common go projects?
Can't we use like https://github.com/eclipse/che-devfile-registry/blob/master/devfiles/go/devfile.yaml#L28 ?
Or if needed introduce one more into https://github.com/eclipse/che-dockerfiles

Reply:
I agree that it's probably easier to create a new `che-dockerfiles` repo and push the dev container there. However, one benefit of doing everything in this repo is that it's easier to keep our development image in sync with the repo, if e.g. we move to go modules and no longer need `dep`.

Reply:
>  I can see a case a year in the future where we've forgotten to remove dep from a che-dockerfiles image, and nobody knows whether it's still required.

Should we try to build two go dev images, like dep based, go modules based...

I tried to think if it can work with WorkspaceController... Seems it doesn't since it also requires operator framework which is not typical for all go modules images...
So, I'm OK with having che-plugin-broker-dev at least for time being.

Reply:
Well my thinking with hosting the dockerfile in this repo is that we would keep it in sync with the current requirements -- for now it works for plugin broker; once we move to modules, I imagine that PR would also update the dockerfile to be automatically configured for development.